Portfolio/src/App.vue

```diff
@@ -101,23 +100,39 @@ body {
 .navbar {
     display: flex;
     justify-content: flex-end;
-    margin: 4rem 5rem;
-    font-size: 43px;
+    font-size: 25px;
```

Comment:
We generally see font-sizes as css variables too. You could define those in the separate style file too - could be your next task for example - good to get these things in order before doing too much more actual development as the cleanup work will just continue to grow.

Reply:
We decided that we should do it in a separate branch. We're going to raise a different PR for it, so I'm resolving this comment for now.